client/asset/dcr/dcr_test.go

```diff
@@ -182,38 +182,39 @@ func signFunc(msgTx *wire.MsgTx, scriptSize int) (*wire.MsgTx, bool, error) {
 }
 
 type tRPCClient struct {
-	sendRawHash    *chainhash.Hash
-	sendRawErr     error
-	sentRawTx      *wire.MsgTx
-	txOutRes       map[outPoint]*chainjson.GetTxOutResult
-	txOutErr       error
-	bestBlockErr   error
-	mempoolErr     error
-	rawTxErr       error
-	unspent        []walletjson.ListUnspentResult
-	unspentErr     error
-	balanceResult  *walletjson.GetBalanceResult
-	balanceErr     error
-	lockUnspentErr error
-	changeAddr     stdaddr.Address
-	changeAddrErr  error
-	newAddr        stdaddr.Address
-	newAddrErr     error
-	signFunc       func(tx *wire.MsgTx) (*wire.MsgTx, bool, error)
-	privWIF        *dcrutil.WIF
-	privWIFErr     error
-	walletTx       *walletjson.GetTransactionResult
-	walletTxErr    error
-	lockErr        error
-	passErr        error
-	disconnected   bool
-	rawRes         map[string]json.RawMessage
-	rawErr         map[string]error
-	blockchain     *tBlockchain
-	lluCoins       []walletjson.ListUnspentResult // Returned from ListLockUnspent
-	lockedCoins    []*wire.OutPoint               // Last submitted to LockUnspent
-	listLockedErr  error
-	estFeeErr      error
+	sendRawHash           *chainhash.Hash
+	sendRawErr            error
+	sentRawTx             *wire.MsgTx
+	txOutRes              map[outPoint]*chainjson.GetTxOutResult
+	txOutErr              error
+	bestBlockErr          error
+	mempoolErr            error
+	rawTxErr              error
+	unspent               []walletjson.ListUnspentResult
+	unspentErr            error
+	balanceResult         *walletjson.GetBalanceResult
+	balanceErr            error
+	lockUnspentErr        error
+	changeAddr            stdaddr.Address
+	changeAddrErr         error
+	newAddr               stdaddr.Address
+	newAddrErr            error
+	signFunc              func(tx *wire.MsgTx) (*wire.MsgTx, bool, error)
+	privWIF               *dcrutil.WIF
+	privWIFErr            error
+	walletTx              *walletjson.GetTransactionResult
+	walletTxErr           error
+	lockErr               error
+	passErr               error
+	disconnected          bool
+	rawRes                map[string]json.RawMessage
+	rawErr                map[string]error
+	blockchain            *tBlockchain
+	lluCoins              []walletjson.ListUnspentResult // Returned from ListLockUnspent
+	lockedCoins           []*wire.OutPoint               // Last submitted to LockUnspent
+	listLockedErr         error
+	estFeeErr             error
+	abandonTransactionErr error
```

Comment:
How about `abandonTxErr` so we don't blow up the diff?

Reply:
shortened